src/commcare_cloud/ansible/deploy_hq.yml

```diff
@@ -44,23 +44,22 @@
       vars:
         if_not_done: yarn_install
 
-- name: Run webpack
+- name: Collect static files
   hosts: [webworkers, proxy]
   any_errors_fatal: true
   tasks:
-    - include_role: {name: deploy_hq}
+    - import_role: {name: deploy_hq}
       vars:
-        if_not_done: run_webpack
-      when: use_webpack | default(false)
-  
+        if_not_done: staticfiles_collect
 
-- name: Collect static files
+- name: Webpack Build
```

Comment:
it has to come after collect static files because the `staticfiles` directory is removed.
`build_requirejs` happens after `collectstatic` fyi. when tested on `staging`, all of the files made it to the CDN just fine, if that was the concern?

Reply:
Gotcha, thanks